### DISCUSSION TITLE
Something went wrong checking access: Client network socket disconnected before secure TLS connection was established

### DISCUSSION THREAD

**Discussion by vectorpikachu** [community/community#56541] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

在我更改了vscode中的代理端口以及hosts文件之后，Github Copliot出现了这样的提示：`Something went wrong checking access: Client network socket disconnected before secure TLS connection was established`
并且Github Copilot依然无法使用，请问应该怎么办？十分感谢大家！

After I changed the proxy port in VSCODE and the hosts file, Github Copilot was prompted like this: `Something went wrong checking access: Client network socket disconnected before secure TLS connection was established`
And Github Copilot is still unavailable. What should I do? Thank you so much!

**Comment by mrepol742** [comment, marked as answer]
Try this troubleshooting guide:
https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-your-environment?tool=vscode

**Reply by vectorpikachu** [reply]
对不起，最近我稍微有点忙，今天才注意到了您的回复，可是我打开了这个log文件后，鉴于我对于这方面内容的无知，我还是不能理解这个文件到底有什么含义，如果您能帮助我，我会非常感谢！

I'm sorry, I've been a little busy lately, and I didn't notice your reply until today, but after I opened this log file, due to my ignorance in this area, i still can't understand the meaning of this file, if you can help me, I will be very grateful!

The log file is here:
```
## Configuration

- HTTP proxy: https://127.0.0.1:33210
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: on

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Self Signed Certificates: disabled
- Send Restricted Telemetry: enabled

## Extensions

- Copilot Version: 1.86.82
- Is `win-ca` installed?: true
- Is `mac-ca` installed?: true

## Node setup

- Number of root certificates: 131
- Operating system: Windows_NT
- Operating system version: 10.0.22621
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: Hostname/IP does not match certificate's altnames: Host: default.exp-tas.com. is not in the cert's altnames: DNS:*.media.microsoftstream.com, DNS:*.azureedge.net, DNS:*.origin.mediaservices.windows.net, DNS:*.streaming.mediaservices.windows.net
```

**Reply by vectorpikachu** [reply]
事实上我对于网络之类的知识感到十分迷茫、沮丧，真的谢谢你！

In fact, I feel very confused and frustrated about the knowledge of the network, really thank you!

**Reply by vectorpikachu** [reply]
对不起，我发现这条报错来自于github copilot labs!但是为什么github copilot labs会无法使用呢？

Sorry, I found this error comes from github copilot labs! But why it prints this error? (the error means the title above)
